spec/adb_spec.rb

```diff
@@ -1,10 +1,8 @@
-$LOAD_PATH.unshift('./lib/')
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.